### PR TITLE
feat: add all shards download & unzip script

### PR DESCRIPTION
**Pull request by Luodian**
1. feat: add all shards download & unzip script for no_facev2 and no_face_corev3
2. update corresponding readme to execute the command 

> written with the help of GPT-4, and checked validity by luodian (drluodian@gmail.com)

### CODE REVIEW

File: scripts/download_images.py
Comment (VegB) [inline scripts/download_images.py:1]: Might need to update/revert the changes as there are some more recent edits on this file

**Review by VegB — CHANGES_REQUESTED** [review]
Hi Luodian,

Thank you so much for providing the downloading scripts!! 
I left a few comments for minor changes, would you mind solving them before we merge it to the main stream?

File: scripts/download_images.py
Comment (Luodian) [reply scripts/download_images.py:1]: > Hi Luodian,
> 
> Thank you so much for providing the downloading scripts!! I left a few comments for minor changes, would you mind solving them before we merge it to the main stream?

I alddressed the requested changes. There for making in total 9 commits, if it bothers, you can squash them and then merge.

**Comment by Luodian** [comment]
I already checked the scripts that they can download the mmc4  zipped jsonls, and then use `download_image.py` to download corresponding images then use [convertion script](https://github.com/mlfoundations/open_flamingo/blob/main/open_flamingo/train/convert_mmc4_to_wds.py) to create the required shards file for training MMC4 on Openflamingo.

**Review by VegB — APPROVED** [review]

**Comment by jmhessel** [comment]
Thanks @Luodian and @VegB !

**Pull request merged**